llvm/tools/repo2obj/R2OELFSymbolTable.h

```diff
@@ -318,6 +323,8 @@ auto SymbolTable<ELFT>::insertSymbol(pstore::indirect_string const &Name,
     // definition which wouldn't be right.
     V.Target = Target;
   }
+  if (V.IsWeak && !IsWeak)
+    V.IsWeak = IsWeak;
```

Comment:
Isn’t this simply: `V.IsWeak = V.IsWeak && IsWeak`?

Reply:
Yes, fixed.